### ISSUE TITLE
Creating a navigation bar

### ISSUE THREAD

**Issue by AAllport** [assemble/assemble#952]
## version 
"assemble": "^0.24.3"

## description
I am currently working on the navigation bar.
```html
{{#pages}}
    <ul>
        {{#each items}}
            <li>
                {{this.data.title}}
                {{../../this.title}}
                {{#is this.data.title ../../this.title}}active{{/is}}
            </li>
        {{/each}}
    </ul>
{{/pages}}
```
I am expecting:
`Page1 Page1 active Page2 Page1`
However i am seeing:
`Page1 Page 1 [Object object] Page2 Page1 [Object object]`
There is no errors on the cli
If anyone could provide any suggestion, I would be most appreciative

## assemblefile.js

```js
'use strict';

const sass = require('gulp-sass');
const extname = require('gulp-extname');
const watch = require('base-watch');
const browserSync = require('browser-sync').create();
const clean = require('gulp-clean');

const htmlmin = require('gulp-htmlmin');
const imagemin = require('gulp-imagemin');
const jsmin = require('gulp-uglify');

const assemble = require('assemble');
const app = assemble();
const dir = {src: './src', out: './dist'};
dir.srcAssets = dir.src + '/assets';
dir.outAssets = dir.out + '/assets';


app.use(watch());

app.helper('log', function(context) {
    console.log(context);
});

app.task('load', function (cb) {
    app.data('title', '--REDACTED--');
    app.data('analyticsID', '--REDACTED--');

    app.layouts(dir.src + '/layouts/*.hbs');
    app.pages(dir.src + '/templates/*.hbs', {layout: 'default'});
    cb();
});

app.task('html', function () {
    return app.toStream('pages')
        .pipe(app.renderFile())
        .pipe(extname('.html'))
        .pipe(htmlmin())
        .pipe(app.dest(dir.out))
        .pipe(browserSync.stream());
});```

**Comment by assemblebot** [comment]
@AAllport Thanks for the issue! If you're reporting a bug, please be sure to include:

- The version of `assemble` you are using.
- Your assemblefile.js (This can be in a gist)
- The commandline output. (Screenshot or gist is fine)
- What you expected to happen instead.

If your issue is related to one of the following, please open an issue there:
  - [grunt-assemble](https://github.com/assemble/grunt-assemble/issues/new) Issues with using assemble in grunt or the grunt-assemble library.
  - [handlebars-helpers](https://github.com/assemble/handlebars-helpers/issues/new) Issues with using handlebars helpers from the handlebars-helpers library.

**Comment by criticalmash** [comment]
I noticed your template uses the `{#is}` helper from handlebars-helpers but I don't see handlebars-helpers being required in your assemblefile. That might be what's causing it to output [object].

BTW: if your looking to add auto generated navigation to your site, I wrote a middleware for that https://github.com/criticalmash/assemble-navigation It's still listed as a beta project, but it's actually pretty robust and has been used on a production site with 200+ pages. Check it out. One of features are `isActive` and `isCurrentPage` variables that can be used to highlight menu items in the active path and the current page.
